src/LuaTransformer.ts

```diff
@@ -1218,7 +1361,11 @@ export class LuaTransformer {
 
         return tstl.createExpressionStatement(this.transformExpression(expression));
     }
-
```

Comment:
I know it seems minor, but spacing between functions helps a lot with readability

Reply:
Done